git_machete/git_operations.py

```diff
@@ -144,6 +144,7 @@ def of(value: str) -> Optional["FullTreeHash"]:
 
 
 ReflogEntry = Tuple[FullCommitHash, str]
+ForkPointOverrideData = Tuple[FullCommitHash, FullCommitHash]
```

Comment:
I'd rather define a class (could be a `@dataclass`, if it weren't for 3.6 compatibility...), with named attributes: `to` and `whileDescendantOf`